pipeline/pipeline/calculate_metrics_file.py

```diff
@@ -133,52 +90,12 @@ def calculate_metrics(
         - df["delta.other"]
     )
 
-    # cubic splines
-    df["spline.active"] = cubic_spline(df["delta.active"])
-    df["spline.deceased"] = cubic_spline(df["delta.deceased"])
-    df["spline.hospitalized"] = cubic_spline(df["delta.hospitalized"])
-    df["spline.recovered"] = cubic_spline(df["delta.recovered"])
-
-
-    # Create new columns for non imputed data and delete the temp column
-
-    #daily data
-
-    df["delta.tested.non.impu"] = df["delta.tested.non.impu.temp"]
-    df["delta.confirmed.non.impu"] = df["delta.confirmed.non.impu.temp"]
-    df["delta.deceased.non.impu"] = df["delta.deceased.non.impu.temp"]
-    df["delta.recovered.non.impu"] = df["delta.recovered.non.impu.temp"]
-    df["delta.other.non.impu"] = df["delta.other.non.impu.temp"]
-
-    df=df.drop(columns=["delta.tested.non.impu.temp","delta.confirmed.non.impu.temp","delta.deceased.non.impu.temp","delta.recovered.non.impu.temp","delta.other.non.impu.temp"])
-
-
-    #total column
-
-    df["total.tested.non.impu"] = df["delta.tested.non.impu"].cumsum()
-    df["total.confirmed.non.impu"] = df["delta.confirmed.non.impu"].cumsum()
-    df["total.recovered.non.impu"] = df["delta.recovered.non.impu"].cumsum()
-    df["total.deceased.non.impu"] = df["delta.deceased.non.impu"].cumsum()
-    df["total.other.non.impu"] = df["delta.other.non.impu"].cumsum()
-
-    # active cases by day non imputed
-    df["delta.active.non.impu"] = (
-            df["total.confirmed.non.impu"]
-            - df["total.deceased.non.impu"]
-            - df["total.recovered.non.impu"]
-            - df["delta.other.non.impu"]
-    )
-
-    #21 day moving average
-
-
-    df["MA.21.daily.active.non.impu"] = df["delta.active.non.impu"].rolling(window=21).mean()
-    df["MA.21.daily.deceased.non.impu"] = df["delta.deceased.non.impu"].rolling(window=21).mean()
-    #Hospitilasation function is picking data from imputed columns, hence will be created once imputed is replaced by non imputed
-    #df["MA.21.daily.hospitalized"] = df["delta.hospitalized"].rolling(window=21).mean()
-    df["MA.21.daily.recovered.non.impu"] = df["delta.recovered.non.impu"].rolling(window=21).mean()
-
+    df["MA.21.daily.active"] = df["delta.active"].rolling(window=21).mean()
+    df["MA.21.daily.deceased"] = df["delta.deceased"].rolling(window=21).mean()
+    df["MA.21.daily.hospitalized"] = df["delta.hospitalized"].rolling(window=21).mean()
+    df["MA.21.daily.recovered"] = df["delta.recovered"].rolling(window=21).mean()
```

Comment:
This assumes a proper sort of the data on input. Might want to sort on 'date' just to avoid that assumption

Reply:
Fair point! strangely it never caused any error. I will make the changes